docs/clients/index.md

```diff
@@ -140,6 +140,7 @@ for more information.
 | [CodeDeploy](./code-deploy.md)              | [async-aws/code-deploy](https://packagist.org/packages/async-aws/code-deploy)
 | [CognitoIdentityProvider](./cognito-idp.md) | [async-aws/cognito-identity-provider](https://packagist.org/packages/async-aws/cognito-identity-provider)
 | [DynamoDb](./dynamodb.md)                   | [async-aws/dynamo-db](https://packagist.org/packages/async-aws/dynamo-db)
+| [Ecr](./ecr.md)            | [async-aws/eecr](https://packagist.org/packages/async-aws/ecr)
```

Comment:
Isn't it `ECR` with capital letters?

Reply:
yeah, but i copied `Iam` from below. let me know what you want to do!

Reply:
Yeah. I saw that IAM is also wrong. 

but STS, S3; SQS, SES etc is correct. 

I would like you to change to ECR and we can change Iam to IAM in a separate PR.

Reply:
Done!